.github/workflows/all_solutions.yml

```diff
@@ -518,3 +518,26 @@ jobs:
             C:\IntegrationTestWorkingDirectory\TestResults\**\*TestResults.xml
           if-no-files-found: error
           
+  combine-integration-test-results:
+    needs: run-integration-tests
+    if: ${{ always() }}
+    name: Combine Integration Test Results
+        
+    runs-on: windows-2019
+        
+    steps:
+      - name: Download all workflow run artifacts
+        uses: actions/download-artifact@v2
+        
+      - name: Combine Integration Results
```

Comment:
You should be able to have the upload action gather these for you using wildcard, similar to jenkins.  This would avoid an extra step.

Reply:
My goal is to get the .xml XUnit test results file into a single flattened directory.